Proxy/InternetAdressRequester.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Decorator
+{
+    class InternetAdressRequester : IRequester
```

Comment:
You shouldn't create your own class. Just use base class [HttpWebRequest](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpwebrequest?view=netcore-3.1)

Reply:
done